Devops Content Update (#1454)
kamranahmedse/developer-roadmap: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/121-sed.md

> # Sed
> SED(**S**tream **Ed**itor) command in UNIX can perform lots of functions on file like searching, find and replace, insertion or deletion. By using SED you can edit files even without opening them in editors like [VI Editor](https://www.redhat.com/sysadmin/introduction-vi-editor).
> 
> **Syntax**:
> 
> `$ sed [options].. [script] [input-file]`
> 
> e.g. *$ sed 's/search-regex/replacement-txt/g' file.txt*
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.geeksforgeeks.org/sed-command-in-linux-unix-with-examples/'>What is SED? with examples</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Manual' href='https://www.gnu.org/software/sed/manual/sed.html'>Detailed Manual</BadgeLink>

Reviewer: ```suggestion
# sed

`sed`(**S**tream **Ed**itor) command in UNIX can perform lots of functions on file like searching, finding and replacing, insertion or deletion. By using SED you can edit files even without opening them in editors like [VI Editor](https://www.redhat.com/sysadmin/introduction-vi-editor).

It has the following syntax:

`$ sed [options].. [script] [input-file]` e.g. `$ sed 's/search-regex/replacement-txt/g' file.txt`

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.geeksforgeeks.org/sed-command-in-linux-unix-with-examples/'>What is SED? with examples</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Manual' href='https://www.gnu.org/software/sed/manual/sed.html'>Detailed Manual</BadgeLink>
```
